res/skins/LateNight/style.qss

```diff
@@ -1383,160 +1411,177 @@ QTableView::item:selected {
 QPushButton#LibraryPreviewButton {
```

Comment:
I suggest to add the "hower" feature from Tango for the "preview Button in the library":
```
 /* Button in library "Preview" column */
 QPushButton#LibraryPreviewButton {
-  width: 23px;
-  height: 12px;
-  background-color: transparent;
-  border: 0px;
-  margin: 0px;
-  padding: 0px;
-  }
-  QPushButton#LibraryPreviewButton:!checked{
-    image: url(skin:/style/library_preview_play.png);
-  }
-  QPushButton#LibraryPreviewButton:checked{
-    image: url(skin:/style/library_preview_pause.png);
-  }
+ width: 23px;
+ height: 12px;
+ background: transparent;
+ border: 1px solid transparent;
+ margin: 0px;
+ padding: 0px;
+}
+QPushButton#LibraryPreviewButton:hover,
+QPushButton#LibraryPreviewButton:checked:hover {
+   border: 1px solid #eeeeee;
+}
+QPushButton#LibraryPreviewButton:!checked {
+ image: url(skin:/style/library_preview_play.png);
+ background-color: #0f0f0f;
+}
+QPushButton#LibraryPreviewButton:checked {
+ image: url(skin:/style/library_preview_pause.png);
+ background-color: #000;
+ border: 1px solid #ff6600;
+}
```

before:
![before](https://user-images.githubusercontent.com/3403218/40566447-f277204c-6070-11e8-8551-3a6f01ad31c9.png)
after:
![after](https://user-images.githubusercontent.com/3403218/40566450-f4e90dfe-6070-11e8-979d-acc036ed3073.png)

Reply:
I don't want to add hover effects to LateNight because there aren't any in the skin (except some slight effect for beatsize & AutoDJ spinboxes)

Thanks for pointing me to that OS theme shining through the Preview button, I edited those.
That's how it looks like mow:
![2 1 1-skin-fixes-latenight-preview-bpmedit](https://user-images.githubusercontent.com/5934199/40578387-9630d438-6113-11e8-9ca8-d0a7ba022ba9.png)

![2 1 1-skin-fixes-tango-preview-bpmedit](https://user-images.githubusercontent.com/5934199/40578391-9a66bc7a-6113-11e8-9b43-0fab9fd845ed.png)

That small overlap at the BPM spinbox cannot be removed for some reason, but at least it's easy to read now, independent from OS theme

Reply:
>I don't want to add hover effects to LateNight

I think hover for the whole selected line would be an improvement in usability!

>OS theme shining through the Preview button

indeed better

Maybe you could change the highlighting of the playlist/crates which also contain the currently selected track from "just" bold to another color (playlist "dg" in the picture) to improve visibility of this feature
![playlist_contain_track_highlight](https://user-images.githubusercontent.com/3403218/40579598-aacb4654-612b-11e8-8690-d395c2c8b91f.png)